### ISSUE TITLE
glfwWindowShouldClose: Assertion `window != NULL' failed.

### ISSUE THREAD

**Issue by leoncyao** [dilevin/CSC2549-a1-mass-spring-1d#3]
I tried to compile the starter code on teach.cs through a remote login and got this error. 

![image](https://user-images.githubusercontent.com/35909284/93393852-13a23800-f828-11ea-87a2-0ba424481103.png)

I was using the NX client to login remotely. Did anybody else get something similar? I am also taking CSC419 and this happens when I compile their starter code for the first assignment, so it might have to do with using the libigl library through a remote connection (or it could just be my computer).
 
Thanks!

**Comment by honglin-c** [comment]
Could you try to run it on your local computer? Using either the command line in osx/Linux or visual studio in windows.

**Comment by leoncyao** [comment]
> Could you try to run it on your local computer? Using either the command line in osx/Linux or visual studio in windows.

Yes it works on a virtual machine running linux, so I suspect libigl has problems with the NX client.

**Comment by honglin-c** [comment]
Yes... Unfortunately, there seem to be some problems with OpenGL when the cdf machines are accessed remotely. I would recommend running the code locally. Sorry for the inconvenience.
